src/olympia/activity/admin.py

```diff
@@ -31,4 +32,52 @@ def has_delete_permission(self, request, obj=None):
         return False
 
 
+class ReviewActionReasonLogAdmin(admin.ModelAdmin):
+    date_hierarchy = 'created'
+    fields = (
+        'created',
+        'activity_log',
+        'user_email',
+        'reason',
+    )
+    list_display = (
+        'created',
+        'activity_log',
+        'reason',
+        'user_email',
+    )
```

Comment:
define `list_select_related` to efficiently preload all the `UserProfile` instances for the emails (or you end up with an query per row in the results table)

Reply:
I added `list_select_related = ('activity_log__user',)`. Is that what you mean here?

Reply:
yes